hubs/static/js/fxos.js

```diff
@@ -0,0 +1,8 @@
+(function (){
+    var btn = document.getElementById("mozinstall");
+    console.log("mozinstall?")
+    btn.addEventListener("click", function(e) {
+        navigator.mozApps.install("http://127.0.0.1:8000/static/manifest.webapp");
```

Comment:
We can't really hardcode this address here.  It needs to be somehow configurable.